cmd/private-org-sync/main.go

```diff
@@ -38,42 +38,36 @@ type options struct {
 
 func (o *options) validate() []error {
 	var errs []error
-	if o.configDir == "" {
-		errs = append(errs, fmt.Errorf("--config-dir is required"))
-	}
 
+	// allow users to pass the previous set of flags
+	if o.Org == "" {
+		o.Org = o.org
+	}
+	if o.Repo == "" {
+		o.Repo = o.repo
+	}
```

Comment:
The previous option had a different semantics -- it expected a full `org/repo` option, and it wasn't possible to pass both `--only-repo` and `--only-org` options, so if we want to keep them, they need to be kept this way.

(I don't think we necessarily need to keep the old stuff. Our job only uses `--only-org` so we can keep that for the transitional period and then get rid of it)